code/dal.py

```diff
@@ -94,6 +121,34 @@ def __clean_data(data):
     return data
 
 
+def __clean_data_glove(data):
```

Comment:
how is this related to glove vocabulary? does it mean that this method cannot be used when we train our custom word2vec?

Reply:
The words in GloVe are tokenized using the StanfordParser which behaves a bit differently compared to naive replacing of punctuations and splitting
Eg. can't is tokenized as ca + n't and punctuations have their word embeddings as well

We can definitely use the same for custom word2vec as well
But are not constrained by it

Perhaps the function can be renamed to just __clean_data once we deprecate the old version.